idaplugin/rematch/actions/match.py

```diff
@@ -16,8 +16,7 @@ class MatchAction(base.BoundFileAction):
   name = "&Match"
   dialog = MatchDialog
 
-  def __init__(self, *args, **kwargs):
-    super(MatchAction, self).__init__(*args, **kwargs)
+  def reset(self):
     self._running = False
```

Comment:
I saw that you removed it, I didn't understand why you wanted to add it, thus the question mark.